### PR TITLE
Replace image with intro video

### LINKED ISSUE #597: Script & film workshop welcome video
See also: https://github.com/orgs/oddbird/projects/10/views/1?filterQuery=is%3Aissue+video&pane=issue&itemId=57266354

- [x] script it
- [x] film it
- [x] draft edit
- [x] @SondraE review
- [x] @stacyk review
- [x] @SondraE thumbnail review/approve/make public

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Related Issue(s)
_Reminder to add related issue(s), if available._

Fixes #597

### CODE REVIEW

File: content/workshops/cascading-layouts.md [hunk 1]
```diff
@@ -55,14 +55,7 @@ summary: |
 {% import 'embed.macros.njk' as embed %}
 {% import 'quotes.macros.njk' as quotes %}
 
-{{ embed.figure([{
-  img: 'talks/layout-workshop.png',
```
Comment (jgerigmeyer) [inline content/workshops/cascading-layouts.md:59]: We could remove this image if it's not being used?

**Comment by mirisuzanne** [comment]
> I wonder if we also want to add a blurb (or banner) about the workshop on our homepage temporarily?

I think we should - but I think that's a different card? @SondraE are we already tracking that somewhere?

**Comment by SondraE** [comment]
> I wonder if we also want to add a blurb (or banner) about the workshop on our homepage temporarily?

Yep. Card for that work here: https://github.com/orgs/oddbird/projects/10?pane=issue&itemId=58707445

**Review by SondraE — APPROVED** [review]
LGTM

**Pull request merged**